[SRS] Personas (#113)
arkinmodi/project-sayyara: docs/SRS/SRS.tex

Other stakeholders of the project are the developers, Tiny Coders, who are designing and
implementing the project.

\subsection{Personas}
\begin{enumerate}
	\item Albert Lee is a 25 year old professional working in the accounting industry from Markham, Ontario.
	      He has a wife and three children. Mr. Lee's family two cars: a silver 2005 Toyota Prius LE and a
	      black 2014 Honda CRV. He commutes to Vaughn, Ontario for work. On his way to work each morning, he
	      drops off his children at school. He prefers to drive the Prius for his work commute to save on
	      gas. However, on weekend road trips, grocery shopping, and driving his kids to soccer practice, he
	      prefers to drive the CRV. This is because the car is more spacious and can fit more cargo. Albert

> 	      has a busy schedule, but he adamant in making sure his cars are regularly maintained. This is

Reviewer: ```suggestion
	      has a busy schedule, but he is adamant in making sure his cars are regularly maintained. This is
```

	      his vehicles to have a longer useful-life. Albert is also very cost-conscious and likes to make
	      sure that he is getting the best value for anything he pays for.
	\item David Jones is a 60 year old independent auto repair shop owner in Hamilton, Ontario. He has been
	      operating his repair shop for over 35 years. David currently has 3 employees working for him: one
	      receptionist, and one mechanic, and one apprentice. David splits his time working on administrative
	      tasks, servicing cars, and overlooking the work of his employees. He is a very organized individual
	      and expects the same from his employees. David also believes that customer service is very
	      important, and only by providing the best value and service, can he compete with larger auto repair
	      and maintenance franchises.
	\item Alice Stark is a 22 year old Uber driver. She enjoys working for the platform due to the flexible
	      schedule. Alice drives a 2016 Chevy Cruze. On Uber, she delivers food and provides rides to
	      passengers. The service she offers depends on the time of day. During lunch and dinner hours, she

> 	      prefers to deliver food through UberEats due to the higher volume of orders. She also loves

Reviewer: ```suggestion
	      prefers to deliver food through Uber Eats due to the higher volume of orders. She also loves
```

	      interacting with people and customers, and hearing about their unique life stories. When she is not

> 	      working she enjoys spending her: time meeting her friends at restaurants, hiking, and playing

Reviewer: ```suggestion
	      working, she enjoys spending her: time meeting her friends at restaurants, hiking, and playing
```
